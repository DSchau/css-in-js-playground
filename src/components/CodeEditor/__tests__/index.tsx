import * as React from 'react';
import { shallow } from 'enzyme';

import { CodeEditor } from '../';

test('it matches snapshot', () => {
  expect(shallow(<CodeEditor />)).toMatchSnapshot();
});
