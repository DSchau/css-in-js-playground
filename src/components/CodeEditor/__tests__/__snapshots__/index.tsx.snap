// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<glamorous(div)>
  <glamorous(textarea)
    innerRef={[Function]}
  />
</glamorous(div)>
`;
